source/guides/1.1/getting-started.md

```diff
@@ -1031,7 +1029,7 @@ resources :books, only: [:index, :new, :create]
 ```
 
 To get a sense of what routes are defined, now we've made this change, you can
-use the special command-line task `routes` to inspect the end result:
+run `bundle exec hanami routes` on your command-line to inspect the end result:
```

Comment:
I think it makes sense to tell the reader what they should run in their command-line.